tests/main.spec.js

```diff
@@ -397,5 +398,40 @@ describe('StepZilla', () => {
         expect(enzymeWrapper.find('.progtrckr').childAt(2).hasClass('progtrckr-doing')).to.be.true;
       });
     });
+    
+    describe('onStepChange: not null use case', () => {
+        let onStepChange;
+        let enzymeWrapper;
+        
+        beforeEach(() => {
+            onStepChange = sinon.spy();
+            enzymeWrapper = setup(3, {
+                startAtStep: 1,
+                onStepChange
+            }).enzymeWrapper;
+        })
+    
+        it('should call onStepChange when clicked to next step', (done) => {
+            
+            enzymeWrapper.find('.footer-buttons #next-button').simulate('click');
+    
+            // click above is promise driven so it's async, setTimeout is probably not the best way to do this but it will do for now
+            setTimeout(() => {
```

Comment:
The same method was also used in the test earlier so I just decided to be lazy and use the same method again since it's only a test (it's enough if it works).